frontend/src/diagram/sprotty/utils/geometry.tsx

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
```

Comment:
Yu don't need a tsx file if there are no React elements, use the ts extension

Reply:
done